lib/dartdoc.dart

```diff
@@ -107,6 +107,14 @@ class Dartdoc extends PackageBuilder {
     outputDir = Directory(config.output)..createSync(recursive: true);
   }
 
+  /// An asynchronous factory method that builds Dartdoc's file writers
+  /// and returns a Dartdoc object with them.
+  @deprecated
+  static Future<Dartdoc> withDefaultGenerators(
```

Comment:
optionally, a comment pointing the user to fromContext seems like a good idea.

Reply:
Done